.github/workflows/merge-main.yml

```diff
@@ -32,7 +32,7 @@ jobs:
             oc_version: "4.13"
             overwrite: true
             parameters:
-              -p OC_NAMESPACE=a4b31c-test
+              -p OC_NAMESPACE=${{ vars.OC_NAMESPACE }}
```

Comment:
I think this needs to be test and the one below needs to be prod (different values), but I didn't look at this closely so could be wrong.

Reply:
This is Github environment variable. I just feel we don't need to hardcode it. But it's unrelated to database restore, I just noticed it. I can change it back if we feel unsafe. But ideally we should use the variable, not hardcode it.
![image](https://github.com/user-attachments/assets/33b4b939-6532-49f6-a102-d9f02f8b58a3)